packages/superset-ui-core/test/callApi/parseResponse.test.js

```diff
@@ -0,0 +1,84 @@
+/* eslint promise/no-callback-in-promise: 'off' */
+import fetchMock from 'fetch-mock';
+import callApi from '../../src/callApi/callApi';
+import parseResponse from '../../src/callApi/parseResponse';
+
+import { LOGIN_GLOB } from '../fixtures/constants';
+import throwIfCalled from '../utils/throwIfCalled';
+
+describe('parseResponse', () => {
```

Comment:
nit
Add `()` in `describe('parseResponse()'`) so we know it is a function when reading test result